docs/actions.md

```diff
@@ -308,13 +310,12 @@ JavaScript functions that run asynchronously may need to return the activation r
 
 
   ```
-  $ wsk activation get b066ca51e68c4d3382df2d8033265db0
+  wsk activation get b066ca51e68c4d3382df2d8033265db0
   ```
- ```
+ ```json
   {
       "start": 1455881628103,
-      "end":   1455881648126,
-      ...
```

Comment:
I think these dots should stay. Without them, it appears as if the activation record only contains the fields `start` and `end`.